src/ui/rangeInput/RangeInput.tsx

```diff
@@ -0,0 +1,9 @@
+import * as Styled from './RangeInput.styles';
+import { RangeInputProps } from './RangeInput.types';
+
+export const RangeInput = (props: React.ComponentPropsWithoutRef<'input'> & RangeInputProps) => (
```

Comment:
Tak się zastanawiam, a może to wrzucić gdzieś do komponentu filtrów? Bo ten input jest dość mocno specyficzny dla tamtego kawałka kodu

Reply:
Mógłby być użyty również do innych filtrów jak np. moc silnika, ale to rzeczywiście i tak w obrębie filtrów by się skończyło. Przeniosłem.